configuration/reports/reportdescriptors/dataexports/sql/msppReport.sql

```diff
@@ -565,6 +567,207 @@ SELECT  (@malaria_test_microscopique_vivax +
          ) INTO @malaria_test_microscopique_total ;
          
 
+-- PHYSICAL VIOLENCE
+SELECT 
+SUM(IF(concept_name(disp.value_coded, 'en') = 'Death' AND p.gender ='F',1,0)),

```

Comment:
it would be better to use the concepts for these dispositions, in case the names ever change.

E.g. use disp.value_coded = concept_from_mapping('PIH',DEATH') instead of  concept_name(disp.value_coded, 'en') = 'Death'

Or actually it would be more efficient to set @death = concept_from_mapping('PIH',DEATH');  and then use "if(disp.value_coded =@death..." or something.  

The mappings are more stable than the names are and should be used.